packages/styles/src/core/layout/map-wrapper/README.md

```diff
@@ -1,119 +1,193 @@
 
-```html
-noSource: true
-responsive: true
----
-<iframe src="/examples/layouts/map-wrapper/map-wrapper-bottom.html" style="width: 100%; height: 100%;">
-```
-
-```code
-lang: html
----
-<body class="as-app">
-  <nav class="as-toolbar-tabs">
-    <span onclick="_showMap(event)" class="as-toolbar-tabs__item as-toolbar-tabs__item--active">MAP</span>
-    <span onclick="_showBottomPanel(event)" class="as-toolbar-tabs__item">PANEL</span>
-  </nav>
 
+```html
+<div class="as-app" style="width:100%; height:300px;">
   <main class="as-app-content">
-    <div class="as-map-wrapper">
-      <div id="map"></div>
-      <div class="as-bottom-bar">
-        Bottom panel content.
+    <div class="as-map-wrapper" style="background: #B5E0F9;">
+      <!-- CARTO.js or CARTO-VL will place the map on this div  -->
+      <div id="map"></div> 
+      <div class="as-bottom-bar" style="background: #F8E71C; height: 100px;">
+        .as-bottom-bar
       </div>
     </div>
   </main>
-</body>
+</div>
 ```
 
-### Map wrapper + Floating panels
-
-To add floating panels to the map area, to use them for legends for example, you need to add first an element with class `as-panels` inside the `as-map` element.
+## .as-panels
 
-Then you need to add as much elements you want to contains the actual floating panels. The possible positions are set with a combination of place modifiers. For instance, if we want a container to place legends at the top right corner, you need to write:
+Use this class as a child of the `as-map-wrapper` element. The `as-panels` wrapper covers the map area but wont interfer the UI. This element should only be used as a container for the panels.
 
-```code
-lang: html
----
-<div class="as-panels">
-  <div class="as-panel as-panel--top as-panel--right">
-    <div class="as-panel__element">
-      Top Right legend
-    </div>
-    <div class="as-panel__element">
-      Some other content at Top Right
+```html
+<div class="as-app" style="width:100%; height:300px;">
+  <main class="as-app-content">
+    <div class="as-map-wrapper" style="background: #B5E0F9;">
+      <!-- CARTO.js or CARTO-VL will place the map on this div  -->
+      <div id="map"></div> 
+      <div class="as-panels" style="background: rgba(0, 0, 0, 0.5);">
+        as panels
+      </div>
     </div>
-  </div>
+  </main>
 </div>
-
 ```
 
-The possible modifiers for positioning are:
+## .as-panel
+
+An `as-panel` element is a container for the `as-panel__element` that allows to specify the position and how the panel elements will be stacked.
+
+You can combine two of the following class modifiers to specify the panel position:
+
 - `as-panel--top`
 - `as-panel--middle`
 - `as-panel--bottom`
 - `as-panel--left`
 - `as-panel--center`
 - `as-panel--right`
 
-You can add an aditional modifier class to allow the panels grow vertically. This way, Airship will stack `as-panel__element` one above the other.
```

Comment:
Did we remove `.as-panel--vertical`?

Reply:
nope, but it wasn't documented, good catch :D

Reply:
nope, is documented below